repository-data/application/src/main/resources/hcm-config/namespaces/website.cnd

```diff
@@ -57,6 +57,9 @@
 [website:award] > hippo:compound, hippostd:relaxed
   orderable
 
+[website:bannerTargeted] > hippostd:relaxed, hippotranslation:translated, website:basedocument
+  orderable
```

Comment:
I didn't see this 'bannerTargeted' anywhere else in the PR. Is it meant to be here?

Reply:
it is, inside website folder, targetedBanner is create to be added in the catalog and put in the shared-container

Reply:
Sorry I got confused with the container. I have erased the website:bannerTargeted